src/app/containers/App/App.jsx

```diff
@@ -31,6 +31,20 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
   });
 
   const isInitialMount = useRef(true);
+  const shouldSetFocus = useRef(false);
+  const focusContentEl = () => {
```

Comment:
I just realised this function will be recreated on every render so I'd just put the contents of it inside of the useEffect e.g.

```js
  useEffect(() => {
    if (shouldSetFocus.current) {
      const contentEl = document.querySelector('h1#content');
      if (contentEl) {
        contentEl.focus();
      }
      shouldSetFocus.current = false;
    }
  }, [state.loading]);
```